controllers/services/monitoring/monitoring_controller_actions.go

```diff
@@ -111,6 +112,17 @@ func updateStatus(ctx context.Context, rr *odhtypes.ReconciliationRequest) error
 	if !ok {
 		return errors.New("instance is not of type *services.Monitoring")
 	}
+
+	// TODO: deprecate phase
+	m.Status.Phase = "NotReady"
+	// condition
+	nc := metav1.Condition{
+		Type:    string(ReadyConditionType),
+		Status:  metav1.ConditionFalse,
+		Reason:  status.ReconcileInit,
```

Comment:
I'm not sure if `ReconcileInit` makes any sense to be honest, so if the deployment won't get up and running, then we would leave the reason to `ReconcileInit` forever

Reply:
updated not usre if it is good to use 
```
                Type:    string(ReadyConditionType),
		Status:  metav1.ConditionFalse,
		Reason:  status.PhaseNotReady,
		Message: "Prometheus deployment is not ready",
	
```